coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -233,9 +251,9 @@ private[filodb] final class NodeCoordinatorActor(metaStore: MetaStore,
 
   private def reset(origin: ActorRef): Unit = {
     ingesters.values.foreach(_ ! PoisonPill)
-    queryActors.values.foreach(_ ! PoisonPill)
+    tempQueryRouter.queryActors.values.foreach(_ ! PoisonPill)
     ingesters.clear()
-    queryActors.clear()
+    tempQueryRouter.queryActors.clear()
```

Comment:
We should not do this. Only the parent needs to stop the child actors.

Reply:
Fixed in 7eec2ee